airbrakes/data_handling/apogee_predictor.py

```diff
@@ -117,7 +117,7 @@ def _curve_fit(self) -> npt.NDArray[np.float64]:
             self._curve_fit_function,
             self._cumulative_time_differences,
             self._accelerations,
-            p0=CURVE_FIT_INITIAL,
+            p0=ApogeePredictorSettings.CURVE_FIT_INITIAL.value,
```

Comment:
Similar to my previous comment, this should not be an enum. This kinda misinterprets the idea/use case of enums. 

They should be used when you have a place that can only accept a set of specific values. E.g. you could have a compass class that currently represents the direction it is facing with an enum for direction containing the values North, South, East, and West.

In this case, it doesn't make sense to use an enum as 1. we only have two values, but 2. (and more importantly) the place that this is being used will only ever use this one specific constant. It won't ever use other values from the apogee predictor settings.